cdc/sinkv2/eventsink/mq/worker.go

```diff
@@ -289,6 +298,48 @@ func (w *worker) asyncSend(
 	return nil
 }
 
+func (w *worker) sendMessages(ctx context.Context) error {
+	inputCh := w.encoderGroup.Output()
```

Comment:
Why call it `input` rather than `output`?

Reply:
It's encoder group's output, but `sendMessages` input.